### PR TITLE
Fixes the make targets and runs the tests for kba

### PR DESCRIPTION
**Pull request by faiq**
- uses file make goals instead of a wrapper phony goal 
- fixes the find in the test script so KBA tests can run too
- adds blank test for ambassador to make KBA tests to pass

### CODE REVIEW

**Review by joejulian — APPROVED** [review]

**Review by joejulian — APPROVED** [review]

**Review by cprovencher — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -50,8 +48,6 @@ $(KUBEADDONS_TEST_KUBECONFIG): install-bin
 .PHONY: kind-test
 kind-test: create-kind-cluster
 	KUBEADDONS_REPO=$(KUBEADDONS_REPO) TESTING_BRANCH=$(TESTING_BRANCH) KUBEADDONS_TEST_KUBECONFIG=$(KUBEADDONS_TEST_KUBECONFIG) $(ROOT_DIR)/run-tests.sh
-	KUBECONFIG=$(KUBEADDONS_TEST_KUBECONFIG) bin/kind delete cluster
```
Comment (orsenthil) [inline Makefile:53]: Why was this (cleanup) not required?

**Review by orsenthil — APPROVED** [review]
LGTM

File: Makefile (hunk 1 above)
Comment (orsenthil) [reply Makefile:53]: I gave LGTM thinking, there is a good reason for this removal, but not obvious to me.

Please share the details before merge.

Comment (faiq) [reply Makefile:53]: This stuff runs in the make clean goal

**Pull request merged**